### DISCUSSION TITLE
Delete pattern + confirmation message pattern

### DISCUSSION THREAD

**Discussion by mma1504** [twilio-labs/paste#3278] [Office hours] [answered]
### Background

I'm working on a new content pattern for Paste around confirmation message patterns. There is already a delete pattern, which it might make sense to roll into the broader confirmation message pattern.

### Feedback

I'm hoping to get feedback from the Paste team on whether this seems like a reasonable course of action and whether there are any other considerations I should take into account before possibly going down this path.

### Stage of design

lo-fi

### Are you looking for visual design support?

- [ ] Yes

**Comment by serifluous** [comment, marked as answer]
Thanks for coming to office hours! You can find the recording here: https://www.loom.com/share/d0f4dfdaf4734f219efb8ea079f7883d?sid=5ebbfe50-1357-4b41-ab4b-69fec7b88e4b and the password in our slack channel
